backend/dal/Migrations/v01.14.01/Down/PostDown/02-WorkflowProjectStatus.sql

```diff
@@ -0,0 +1,5 @@
+PRINT 'Delete WorkflowProjectStatus - SPL --> On Hold'
+
+DELETE FROM dbo.[WorkflowProjectStatus]
```

Comment:
We can't remove this unless there are no projects currently using it.  Additionally, why are you deleting it?  It looks like your script below just adds it back?  Is PIMS-54 enabling the ability for "Not in SPL" to be put "On Hold", or "Transferred within the GRE"?

Reply:
I did this with a rollback in mind (I am probably misunderstanding the down scripts). But yes PIMS-54 is to enable Not in SPL to be put On Hold or Transferred within the GRE. The transitions are already enabled and Transferred within the GRE is part of SPL already, I just had to add On Hold to the grouping

Reply:
I think I was looking at the scripts backwards.  Should have noticed this was a down script.